### PR TITLE
style: added utility class

### PR DESCRIPTION
**Pull request by dev-prabhat**

### CODE REVIEW

File: components/badge.css [hunk 1]
```diff
@@ -54,10 +48,6 @@
   height: 1.2em;
   text-align: center;
   color: var(--secondary-theme-color);
-
-  position: absolute;
```
Comment (sanu29) [inline components/badge.css:58]: repeated in same class

**Review by sanu29 — APPROVED** [review]
All good for me. Just a suggestion you can name utile as utility.
extra space could be removed in some places. as well instead of repeating position : absolute in every class you could have made it a class in utilities.

But All good to merge no major changes. These could be done later also

Great work

File: components/badge.css (hunk 1 above)
Comment (dev-prabhat) [reply components/badge.css:58]: I will definitely go through this and made changes accordingly

**Review by srishtigautam21 — APPROVED** [review]
good work

File: components/badge.css (hunk 1 above)
Comment (dev-prabhat) [reply components/badge.css:58]: Actually, that code part is deleted that's why it is shown in red background-color

**Pull request merged**